### DISCUSSION TITLE
how to organize imports?

### DISCUSSION THREAD

**Discussion by Akronae** [rudderlabs/rudder-server#2943] [Q&A] [answered]
I'd like to move some destinations to better view relations between source/dest. 
Could not find any way to do this, is it possible?

![image](https://user-images.githubusercontent.com/17302866/217037219-fc2f2e59-b09e-4b86-90b9-8bb6dfd80134.png)

**Comment by gitcommitshow** [comment, marked as answer]
It is not possible as of now @Akronae 
But this looks like a useful feature. In order to raise a feature request, can you help me with more context on this. How do you expect this feature to work?

**Reply by Akronae** [reply]
I'd just expect any destination/source to be draggrable on its own column! (<=> change source/destination indexes by dragging them) + an auto sort feature would be nice (autosorting algorithm optimizing least connection line overlaps)

**Reply by gitcommitshow** [reply]
Noted. We are working on revamping the connections page. Will consider your feedback. Stay tuned for the updates.
Thank your for the feedback.
